addons/checkparam/data/settings.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.1" ?>
```

Comment:
Should delete this file, see my previous comment on `config.load`

Reply:
done.